github/ci/prow-deploy/files/jobs/kubevirt/project-infra/project-infra-periodics.yaml

```diff
@@ -35,17 +37,12 @@ periodics:
       - --template
       - --ceiling=1
       - --confirm
-      volumeMounts:
-      - name: token
-        mountPath: /etc/github
-    volumes:
-    - name: token
-      secret:
-        secretName:  commenter-oauth-token
```

Comment:
I think that this is a different token than what we are using everywhere else. Can you check please? If that is the case, you might create a separate preset (i.e. `commenter-github-token`)

Reply:
yeah I will double check this.

Reply:
The tokens were different so I added commenter-oauth-token and unprivileged token back in latest update.